spec/imsc-hrm.html

```diff
@@ -11,16 +11,16 @@
     }
   </style>
 <script src="https://www.w3.org/Tools/respec/respec-w3c" class="remove" defer></script>
-</script>
   <script class='remove'>
 var respecConfig = {
-                    specStatus:   "WD"
+                    specStatus:   "CR"
```

Comment:
Weird that this appears in the PR Preview as a Candidate Recommendation Snapshot - my reading of the Process is that the first transition to CR from WD is not a Snapshot, just a CR. @himorin can you confirm what should be showing here for first CR?

Reply:
https://www.w3.org/2021/Process-20211102/#transition-cr
Candidate Recommendation is consisted with CR Snapshot and CR Draft, now.

> The first Candidate Recommendation publication after approval of a [Transition Request](https://www.w3.org/2021/Process-20211102/#trans-req) is always a [Candidate Recommendation Snapshot](https://www.w3.org/2021/Process-20211102/#candidate-recommendation-snapshot). The [Team](https://www.w3.org/2021/Process-20211102/#team) must announce the publication of the [Candidate Recommendation Snapshot](https://www.w3.org/2021/Process-20211102/#candidate-recommendation-snapshot) to other W3C groups and to the public.

Reply:
Ah, thank you!